gacl/gacl.class.php

```diff
@@ -41,7 +41,11 @@
 
 //openemr configuration file - bm - 05-2009
 // to collect sql database login info and the utf8 flag
-include_once(dirname(__FILE__).'/../library/sqlconf.php');
+// also collect eh adodb libraries to support mysql_log that is needed for mysql ssl support
```

Comment:
typos:
``````
// also collect the adodb libraries to support mysqli_mod that is needed for mysql ssl support